src/pipeline/src/etl/transform/transformer/greptime.rs

```diff
@@ -484,76 +448,59 @@ fn resolve_value(
         };
 
     match value {
-        Value::Null => {}
+        VrlValue::Null => {}
 
-        Value::Int8(_) | Value::Int16(_) | Value::Int32(_) | Value::Int64(_) => {
+        VrlValue::Integer(v) => {
             // safe unwrap after type matched
-            let v = value.as_i64().unwrap();
             resolve_simple_type(ValueData::I64Value(v), column_name, ColumnDataType::Int64)?;
         }
 
-        Value::Uint8(_) | Value::Uint16(_) | Value::Uint32(_) | Value::Uint64(_) => {
-            // safe unwrap after type matched
-            let v = value.as_u64().unwrap();
-            resolve_simple_type(ValueData::U64Value(v), column_name, ColumnDataType::Uint64)?;
-        }
-
-        Value::Float32(_) | Value::Float64(_) => {
+        VrlValue::Float(v) => {
             // safe unwrap after type matched
-            let v = value.as_f64().unwrap();
-            resolve_simple_type(ValueData::F64Value(v), column_name, ColumnDataType::Float64)?;
-        }
-
-        Value::Boolean(v) => {
-            resolve_simple_type(
-                ValueData::BoolValue(v),
-                column_name,
-                ColumnDataType::Boolean,
-            )?;
-        }
-
-        Value::String(v) => {
             resolve_simple_type(
-                ValueData::StringValue(v),
+                ValueData::F64Value(v.into()),
                 column_name,
-                ColumnDataType::String,
+                ColumnDataType::Float64,
             )?;
         }
 
-        Value::Timestamp(Timestamp::Nanosecond(ns)) => {
+        VrlValue::Boolean(v) => {
             resolve_simple_type(
-                ValueData::TimestampNanosecondValue(ns),
+                ValueData::BoolValue(v),
                 column_name,
-                ColumnDataType::TimestampNanosecond,
+                ColumnDataType::Boolean,
             )?;
         }
 
-        Value::Timestamp(Timestamp::Microsecond(us)) => {
+        VrlValue::Bytes(v) => {
             resolve_simple_type(
-                ValueData::TimestampMicrosecondValue(us),
+                ValueData::StringValue(String::from_utf8_lossy(&v).to_string()),
                 column_name,
-                ColumnDataType::TimestampMicrosecond,
+                ColumnDataType::String,
             )?;
         }
 
-        Value::Timestamp(Timestamp::Millisecond(ms)) => {
+        VrlValue::Regex(v) => {
```

Comment:
It would be good to add some logs here, I don't think this should be a reasonable branch.